admin/js/common.js

```diff
@@ -562,13 +565,13 @@ var qTranslateX=function(pg)
 								}
 							}
 						}else{
-							continue;
+							continue;
```

Comment:
Hi Michel, I am just curious, what does cause git to show lines like these to be different? Even spacing seems to be all tabs. It is a bit confusing, but not a big problem, I just would like to understand. Do you know what is different in them?